src/main/java/org/veupathdb/service/eda/data/service/AppsService.java

```diff
@@ -352,6 +353,13 @@ public PostAppsDifferentialabundanceVisualizationsVolcanoplotResponse postAppsDi
       new DifferentialAbundanceStatsResponseStream(processRequest(new DifferentialAbundanceVolcanoplotPlugin(), entity))));
   }
 
+  @DisableJackson
+  @Override
+  public PostAppsDifferentialexpressionVisualizationsVolcanoplotResponse postAppsDifferentialexpressionVisualizationsVolcanoplot(DifferentialExpressionVolcanoplotPostRequest entity) {
+    return wrapPlugin(() -> PostAppsDifferentialexpressionVisualizationsVolcanoplotResponse.respond200WithApplicationJson(
+      new DifferentialExpressionStatsResponseStream(processRequest(new DifferentialExpressionVolcanoplotPlugin(), entity))));
```

Comment:
I'm puzzling where `DifferentialExpressionStatsResponseStream` is defined...?

Is it in a previous PR?

Reply:
`DifferentialExpressionStatsResponseStream` gets automatically generated. The auto-generated files aren't usually tracked in git because there are too many of them. But it'll show up in `eda/generated/model`.